includes/class-gp-custom-locale-reasons.php

```diff
@@ -22,7 +22,34 @@ public static function get_custom_reasons( $locale ) {
 		// ),
 		// )
 		// );
-		$locale_reasons = array();
+		$locale_reasons = array(
+			'it' => array(
+				'Consistenza'                   => array(
```

Comment:
Do we need the arrays to be keyed with a string?

Reply:
The key is what is going to be stored in the database so we should try and find an appropriate English identifier (snake-case)